kyuubi-server/src/main/scala/org/apache/kyuubi/engine/KyuubiApplicationManager.scala

```diff
@@ -58,6 +62,7 @@ class KyuubiApplicationManager extends AbstractService("KyuubiApplicationManager
         case NonFatal(e) => warn(s"Error stopping ${op.getClass.getSimpleName}: ${e.getMessage}")
       }
     }
+    FileUtils.deleteQuietly(tempDirForUpload.toFile)
```

Comment:
```
    FileUtils.deleteQuietly(KyuubiApplicationManager.tempDirForUpload.toFile)
```

and remove
```
import org.apache.kyuubi.engine.KyuubiApplicationManager.tempDirForUpload
```

Reply:
```
if (!(FileUtils.deleteQuietly(Paths.get(resource).toFile)) {
  error(...)
}
```

Reply:
Addressed.
1. prefer the direct way `FileUtils.deleteQuietly(KyuubiApplicationManager.tempDirForUpload.toFile)` with less imports
2. use `FileUtils.deleteDirectory` and log exception instead of `FileUtils.deleteQuietly` which doesn't throw exceptions.